dev/com.ibm.websphere.rest.handler/resources/com/ibm/wsspi/rest/config/resources/CWWKOMessages.nlsprops

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.websphere.rest.handler/resources/com/ibm/wsspi/rest/config/resources/CWWKOMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO Rest Handler
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWKO1500 to CWWKO1529
+
+CWWKO1500_NOT_FOUND=CWWKO1500E: No configured instances of {0} were found.
+CWWKO1500_NOT_FOUND.explanation=The requested configuration element could not be found in the server configuration.
+CWWKO1500_NOT_FOUND.useraction=Verify that the configuration element exists in server configuration and that it is correctly spelled with matching case.
+
+CWWKO1501_INVALID_IDENTIFIER=CWWKO1501E: Unique identifier {0} is not valid. The correct unique identifier might be: {1}.
+CWWKO1501_INVALID_IDENTIFIER.explanation=A configured element with the requested unique identifier was not found. An instance with a different unique identifier was found and is reported in case it is the one that you were looking for.
+CWWKO1501_INVALID_IDENTIFIER.useraction=Determine the correct unique identifier to use. For top level configuration elements with an id specified, the unique identifier is the value of the id. Otherwise, the unique identifier is generated.
+
+CWWKO1502_MULTIPLE_FOUND=CWWKO1502E: A conflicting unique identifier, {0}, is specified or computed for multiple configured instances.
+CWWKO1502_MULTIPLE_FOUND.explanation=Unique identifiers for configured instances must not overlap.
+CWWKO1502_MULTIPLE_FOUND.useraction=Update the server configuration to add or modify the id attribute of configured instances such that they have distinct unique identifiers.
```

Comment:
See my note about ID above

**change**
Update the server configuration to add or modify the id attribute of configured instances such that they have distinct unique identifiers.

**to**
Update the server configuration to add or modify the ID attributes of configured instances so that each unique identifier is used only once.